bin/server/cmd/root.go

```diff
@@ -287,7 +290,11 @@ func NewMain(activeProfile Profile, logger *zap.Logger) (*Main, error) {
 	fmt.Printf("debug=%t\n", debug)
 	fmt.Println("-----Config END-------")
 
-	pgBinDir, err := resources.InstallPostgres(resourceDir, pgDataDir)
+	pgUser := "bb"
```

Comment:
I would consider declaring it in the const section. And put "The name of the database storing metadata is the same as pgUser." there as well

Reply:
Moved to profile instead.